trust-quorum/tests/coordinator.rs

```diff
@@ -114,6 +122,47 @@ impl Model {
         }
     }
 
+    /// If we are currently waiting for shares in this epoch then record the
+    /// responder.
+    ///
+    /// If we've received enough shares to recompute the rack secret then
+    /// transition the model operation to `Prepare` and return `Ok(true)`.
+    pub fn handle_share(
+        &mut self,
+        from: PlatformId,
+        epoch: Epoch,
+    ) -> Result<bool, TestCaseError> {
```

Comment:
use an enum here? in particular it would be nice to indicate why `false` was returned.